style.css

```diff
@@ -7,37 +7,65 @@
 
 body {
     font-family: 'Saira', sans-serif;
+    padding-top: 2em;
+    padding-bottom: 5em;
 }
 
-.guitars {
-    height: 90vh;
+/* Media Query for Mobile Devices */
+@media (max-width: 480px) {
+    .brands {
+        margin: 0;
+    }
 }
 
-.guitar {
-    height: 66vh;
+/* Media Query for low resolution  Tablets, Ipads */
+@media (min-width: 481px) and (max-width: 767px) {
+    .brands {
+        margin: 0 4em;
+    }
+}
+
+/* Media Query for Tablets Ipads portrait mode */
+@media (min-width: 768px) and (max-width: 1024px) {
+    .brands {
+        margin: 0 6em;
+    }
+}
+
+/* Media Query for Laptops and Desktops */
+@media (min-width: 1025px) and (max-width: 1280px) {
+    .brands {
+        margin: 0 8em;
+    }
+}
+
+/* Media Query for Large screens */
+@media (min-width: 1281px) {
+    .brands {
+        margin: 0 10em;
+    }
 }
 
 .guitar,
 .guitars {
-    width: 66%;
     margin: 0 auto;
     box-shadow: #26394d 0 2em 3em 0;
 }
 
+.guitars {
+    height: 90vh;
+}
+
+.guitar {
+    height: 66vh;
+}
 
 /* images section */
 
 .image {
-    position: relative;
+    display: flex;

```

Comment:
burada `relative container -> absolute element` yapisi yerine `flexbox` ve negatif margin kullanarak cozdum yapmaya calistiginiz tasarimi. sizin yolladiginiz versiyonunda hepsi bir hizada degildi, bu sekilde olunca hepsi hem bir hizada, hem de absolute isine girmeden is cozulmus oldu. mumkun oldugunca absolute pozisyonlamadan da kaciniyorum ben islerimde, ama bazen gerekiyor tabii ki, gerektiginde kullanmakta sorun yok bence.